sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -615,15 +612,17 @@ abstract class RapidsShuffleThreadedReaderBase[K, C](
 
         override def next(): (Any, Any) = {
           val fetchTimeStart = System.nanoTime()
-          readBlockedTime = 0
+          var readBlockedTime = 0L
```

Comment:
```suggestion
          val readBlockedTime = 0L
```

Reply:
readBlockedTime is incremented in line 619, so var is required